### PR TITLE
Update supported distros in Readme

### PR DESCRIPTION
**Pull request by techguy16**
Ubuntu 22.04 can run PowerShell, so adding it as supported distro.

# PR Summary
In the Readme, there was no "Ubuntu 22.04" as a supported distro in the table. Ubuntu 22.04 does support PowerShell.

## PR Context
Ubuntu 22.04 users probably thought that PowerShell didn't support their distro so I added it as a row in the table.

### CODE REVIEW

**Review by kilasuit — APPROVED** [review]

**Comment by techguy16** [comment]
Hi can you please review these changes?

File: README.md [hunk 1]
```diff
@@ -30,6 +30,7 @@ You can download and install a PowerShell package for any of the following platf
...
 | [Windows (x64)][corefx-win]                | [.msi][lts-windows-64]  | [.msi][rl-windows-64]   | [.msi][pv-windows-64] | [Instructions][in-windows]    |
 | [Windows (x86)][corefx-win]                | [.msi][lts-windows-86]  | [.msi][rl-windows-86]   | [.msi][pv-windows-86] | [Instructions][in-windows]    |
+| [Ubuntu 22.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu22]     | [.deb][pv-deb]        | [Instructions](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu)   |
 | [Ubuntu 20.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu20]     | [.deb][pv-deb]        | [Instructions][in-ubuntu20]   |
 | [Ubuntu 18.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu18]     | [.deb][pv-deb]        | [Instructions][in-ubuntu18]   |
 | [Ubuntu 16.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu16]     | [.deb][pv-deb]        | [Instructions][in-ubuntu16]   |
```
Comment (TravisEz13) [inline README.md:36]:
```suggestion
```

Ubuntu 16 is not supported in 7.3

File: README.md [hunk 2]
```diff
@@ -73,6 +74,7 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 
 [rl-windows-64]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.0/PowerShell-7.3.0-win-x64.msi
 [rl-windows-86]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.0/PowerShell-7.3.0-win-x86.msi
+[rl-ubuntu22]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.0/powershell_7.3.0-1.deb_amd64.deb
 [rl-ubuntu20]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.0/powershell_7.3.0-1.deb_amd64.deb
 [rl-ubuntu18]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.0/powershell_7.3.0-1.deb_amd64.deb
 [rl-ubuntu16]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.0/powershell_7.3.0-1.deb_amd64.deb
```
Comment (TravisEz13) [inline README.md:80]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -30,9 +30,9 @@ You can download and install a PowerShell package for any of the following platf
 | -------------------------------------------| ------------------------| ------------------------| ----------------------| ------------------------------|
 | [Windows (x64)][corefx-win]                | [.msi][lts-windows-64]  | [.msi][rl-windows-64]   | [.msi][pv-windows-64] | [Instructions][in-windows]    |
 | [Windows (x86)][corefx-win]                | [.msi][lts-windows-86]  | [.msi][rl-windows-86]   | [.msi][pv-windows-86] | [Instructions][in-windows]    |
+| [Ubuntu 22.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu22]     | [.deb][pv-deb]        | [Instructions](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu)   |
 | [Ubuntu 20.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu20]     | [.deb][pv-deb]        | [Instructions][in-ubuntu20]   |
 | [Ubuntu 18.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu18]     | [.deb][pv-deb]        | [Instructions][in-ubuntu18]   |
```
Comment (TravisEz13) [inline README.md:35]:
```suggestion
| [Ubuntu 18.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu18]     | [.deb][pv-deb]        | [Instructions][in-ubuntu18]   |
| [Ubuntu 16.04][corefx-linux] | [.deb][lts-deb] | N/A  | N/A | [Instructions][in-ubuntu16] |
```

Comment (TravisEz13) [inline README.md:35]:
```suggestion
| [Ubuntu 18.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu18]     | [.deb][pv-deb]        | [Instructions][in-ubuntu18]   |
| [Ubuntu 16.04][corefx-linux] | [.deb][lts-deb] | N/A  | N/A | [Instructions][in-ubuntu16] |
```

File: README.md [hunk 4]
```diff
@@ -30,9 +30,10 @@ You can download and install a PowerShell package for any of the following platf
...
-| [Ubuntu 16.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu16]     | [.deb][pv-deb]        | [Instructions][in-ubuntu16]   |
+| [Ubuntu 16.04][corefx-linux] | [.deb][lts-deb] | N/A  | N/A | [Instructions][in-ubuntu16] |
 | [Debian 9][corefx-linux]                   | [.deb][lts-deb]         | [.deb][rl-debian9]      | [.deb][pv-deb]        | [Instructions][in-deb9]       |
```
Comment (TravisEz13) [inline README.md:37]:
```suggestion
```
Debian 9 is not supported is 7.2 and newer
https://github.com/dotnet/core/blob/main/release-notes/6.0/supported-os.md#linux

File: README.md [hunk 5]
```diff
@@ -50,7 +50,7 @@ You can download and install a PowerShell package for any of the following platf
 | Platform                 | Downloads (stable)      | Downloads (preview)           | How to Install                |
 | -------------------------| ------------------------| ----------------------------- | ------------------------------|
 | Arch Linux               |                         |                               | [Instructions][in-archlinux]  |
-| Kali Linux               | [.deb][rl-ubuntu16]     | [.deb][pv-deb]           | [Instructions][in-kali]       |
+| Kali Linux               | [.deb][rl-deb.    ]     | [.deb][pv-deb].               | [Instructions][in-kali]       |
```
Comment (TravisEz13) [inline README.md:53]:
```suggestion
| Kali Linux               | [.deb][rl-deb]          | [.deb][pv-deb]                | [Instructions][in-kali]       |
```

**Review by TravisEz13 — APPROVED** [review]

File: README.md [hunk 6]
```diff
@@ -73,10 +73,10 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 
 [rl-windows-64]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/PowerShell-7.3.2-win-x64.msi
 [rl-windows-86]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/PowerShell-7.3.2-win-x86.msi
+[rl-deb]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.0/powershell_7.3.2-1.deb_amd64.deb
```
Comment (TravisEz13) [inline README.md:76]:
```suggestion
[rl-deb]: https://github.com/PowerShell/PowerShell/releases/download/v7.3.2/powershell_7.3.2-1.deb_amd64.deb
```

File: README.md [hunk 7]
```diff
@@ -30,10 +30,10 @@ You can download and install a PowerShell package for any of the following platf
 | -------------------------------------------| ------------------------| ------------------------| ----------------------| ------------------------------|
 | [Windows (x64)][corefx-win]                | [.msi][lts-windows-64]  | [.msi][rl-windows-64]   | [.msi][pv-windows-64] | [Instructions][in-windows]    |
 | [Windows (x86)][corefx-win]                | [.msi][lts-windows-86]  | [.msi][rl-windows-86]   | [.msi][pv-windows-86] | [Instructions][in-windows]    |
+| [Ubuntu 22.04][corefx-linux]               | [.deb][lts-deb]         | [.deb][rl-ubuntu22]     | [.deb][pv-deb]        | [Instructions](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu)   |
```
Comment (daxian-dbw) [inline README.md:33]: Please create a link with `in-ubuntu22`.

Comment (techguy16) [reply README.md]: Added `in-ubuntu22` link

File: README.md [hunk 8]
```diff
@@ -113,6 +113,7 @@ You can also download the PowerShell binary archives for Windows, macOS and Linu
 [in-ubuntu16]: https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-linux#ubuntu-1604
 [in-ubuntu18]: https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-linux#ubuntu-1804
 [in-ubuntu20]: https://docs.microsoft.com/powershell/scripting/install/installing-powershell-core-on-linux#ubuntu-2004
+[in-ubuntu22]: https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu
```
Comment (daxian-dbw) [inline README.md:116]: Remove the `en-us` part and let the page itself decides which locale to use.
```suggestion
[in-ubuntu22]: https://learn.microsoft.com/powershell/scripting/install/installing-powershell-on-linux?view=powershell-7.3#ubuntu
```

**Review by TravisEz13 — APPROVED** [review]

**Comment by techguy16** [comment]
@microsoft-github-policy-service agree

**Review by JamesWTruher — APPROVED** [review]

**Comment by TravisEz13** [comment]
thanks @techguy16  for your contribution

**Comment by techguy16** [comment]
@TravisEz13 No worries - need any help, just ask 😁

**Pull request merged**